clients/venice-push-job/src/main/java/com/linkedin/venice/hadoop/PushJobSetting.java

```diff
@@ -0,0 +1,159 @@
+  public BufferReplayPolicy validateRemoteReplayPolicy;
+  public boolean suppressEndOfPushMessage;
+  public boolean deferVersionSwap;
+  public boolean extendedSchemaValidityCheckEnabled;
+  /** Refer {@link VenicePushJobConstants#COMPRESSION_METRIC_COLLECTION_ENABLED} **/
+  public boolean compressionMetricCollectionEnabled;
+  /** Refer {@link VenicePushJobConstants#USE_MAPPER_TO_BUILD_DICTIONARY} **/
+  public boolean useMapperToBuildDict;
+  public String useMapperToBuildDictOutputPath;
+  public boolean repushTTLEnabled;
+  // specify time to drop stale records.
+  public long repushTTLStartTimeMs;
+  // HDFS directory to cache RMD schemas
+  public String rmdSchemaDir;
+  public String valueSchemaDir;
+  public String controllerD2ServiceName;
+  public String parentControllerRegionD2ZkHosts;
+  public String childControllerRegionD2ZkHosts;
+  public boolean livenessHeartbeatEnabled;
+  public String livenessHeartbeatStoreName;
+  public boolean multiRegion;
+  public boolean d2Routing;
+  public String targetedRegions;
+  public boolean isTargetedRegionPushEnabled;
+  public boolean postValidationConsumption;
+  public boolean isSystemSchemaReaderEnabled;
+  public String systemSchemaClusterD2ServiceName;
+  public String systemSchemaClusterD2ZKHost;
+  public boolean isZstdDictCreationRequired;
+  public boolean isZstdDictCreationSuccess;
+
+  // Multiple compute engine support
+  public Class<? extends DataWriterComputeJob> dataWriterComputeJobClass;
+
+  // Store-config setting
+  public String clusterName;
+  public Schema storeKeySchema;
+  public boolean isChunkingEnabled;
+  public boolean isRmdChunkingEnabled;
+  public long storeStorageQuota;
+  public boolean isSchemaAutoRegisterFromPushJobEnabled;
+  public CompressionStrategy storeCompressionStrategy;
+  public boolean isStoreWriteComputeEnabled;
+  public boolean isStoreIncrementalPushEnabled;
+  public transient HybridStoreConfig hybridStoreConfig;
+  public transient StoreResponse storeResponse;
+
+  // Topic-properties
+  // Kafka topic for new data push
+  public String topic;
+  /** Version part of the store-version / topic name */
+  public int version;
+  // Kafka topic partition count
+  public int partitionCount;
+  // Kafka url will get from Venice backend for store push
+  public String kafkaUrl;
+  public boolean sslToKafka;
+  public CompressionStrategy topicCompressionStrategy;
+  public String partitionerClass;
+  public Map<String, String> partitionerParams;
+  public int amplificationFactor;
+  public boolean chunkingEnabled;
+  public boolean rmdChunkingEnabled;
+  public String kafkaSourceRegion;
+  public transient RepushInfoResponse repushInfoResponse;
+
+  // Schema-properties
+  public boolean isAvro = true;
+  public int valueSchemaId; // Value schema id retrieved from backend for valueSchemaString
+  public int derivedSchemaId = -1;
+  public String keyField;
+  public String valueField;
+
+  public Schema fileSchema;
```

Comment:
We need both in some place. So, instead of repeated `toString`, I added both to this object since it is expected to be an object that holds state during VPJ job.